python/bindings.cpp

```diff
@@ -1173,13 +1173,69 @@ PYBIND11_MODULE(_libsonata, m) {
 
     bindStorageClass<EdgeStorage>(m, "EdgeStorage", "EdgePopulation");
 
+    PYBIND11_NUMPY_DTYPE(SpikeStruct, node_id, timestamp);
+
     py::class_<SpikeReader::Population>(m, "SpikePopulation", "A population inside a SpikeReader")
         .def("get",
              &SpikeReader::Population::get,
              DOC_SPIKEREADER_POP(get),
              "node_ids"_a = nonstd::nullopt,
              "tstart"_a = nonstd::nullopt,
              "tstop"_a = nonstd::nullopt)
+        .def(
+            "get_array",
```

Comment:
I don't think we need both the `get_array` and the `get_dict`, since they are nearly equivalent.  Let's just go for the `get_dict` one.